src/garage/tf/optimizers/conjugate_gradient_optimizer.py

```diff
@@ -1,27 +1,109 @@
+"""Conjugate Gradient Optimizer.
+
+This optimizer uses conjugate gradient method to compute search direction,
```

Comment:
Please try your best not to use `This foo uses bar` -- Try an imperative statement like "Uses the Conjugate Gradient method..."

Also, for subtle code like this, if you found a good reference for the math, please include a citation with URL for it.